.github/workflows/ci-workflow.yml

```diff
@@ -47,7 +76,6 @@ jobs:
 
   job_unit_test:
     name: Unit test
-    needs: job_build_debug
```

Comment:
It could, but in this case I think unit tests compilation is not that tied to the whole project compilation. These tests are autonomous from the app `Makefile` constraints, in particular the SDK's.
What do you think?

Reply:
Correct.